metabase/metabase
# Add reproduction test

Relates to https://github.com/metabase/metabase/issues/13764

### Description

This adds reproduction test for #13764. The root cause of the issue itself should be fixed in MBQLv2 as we use `ML.available_binning_strategies` lib method to check if a column is binnable.

### How to verify

- Run added unit test. It should pass.
- Try to uncomment final lines. Then test should fail.

### Checklist
